easybuild/framework/easyblock.py

```diff
@@ -2994,24 +2995,13 @@ def sanity_check_rpath(self, rpath_dirs=None):
                 for path in [os.path.join(dirpath, x) for x in os.listdir(dirpath)]:
                     self.log.debug("Sanity checking RPATH for %s", path)
 
-                    out, ec = run_cmd("file %s" % path, simple=False, trace=False)
-                    if ec:
-                        fail_msg = "Failed to run 'file %s': %s" % (path, out)
-                        self.log.warning(fail_msg)
-                        fails.append(fail_msg)
-
-                    # only run ldd/readelf on dynamically linked executables/libraries
-                    # example output:
-                    # ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked (uses shared libs), ...
-                    # ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, not stripped
-                    if "dynamically linked" in out:
+                    out = get_linked_libs_raw(path)
+
+                    if out is None:
+                        self.log.debug("%s is not dynamically linked, so skipping it in RPATH sanity check", path)
```

Comment:
Just so that we don't necessarily lie, it might be that ldd failed right?
```suggestion
                        self.log.debug("ldd failed one %s. Possibly not dynamically linked, so skipping it in RPATH sanity check", path)
```

Reply:
I tweaked the log message differently, should be correct now, see 0e126d256